cpp/tests/copying/utility_tests.cpp

```diff
@@ -18,6 +18,7 @@
 #include <cudf/copying.hpp>
 #include <cudf/strings/detail/utilities.hpp>
 #include <cudf/table/table.hpp>
+#include <cudf/utilities/type_checks.hpp>
```

Comment:
The only function declared in this header is `column_types_equal`, which does not appear to be used in this file. Can we remove this include? https://github.com/rapidsai/cudf/blob/branch-22.08/cpp/include/cudf/utilities/type_checks.hpp